### PR TITLE
Contact feature for Portfolio

### PR DESCRIPTION
**Pull request by hydaline-code**
Hello everyone please here is the updated feature of the portfolio version which is the contact form. I followed the recommendations as follows for this part. I had a pair-program review. Please kindly do the code review so I can proceed to the adjustment for the final submission. Thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,123 +137,155 @@ <h2>Profesional Art Printing Data</h2>
...
+
+            <label for="fname"></label>
+            <input type="text" id="fname"  maxlength = "30" placeholder="Full name" required /><br>
+            <hr class="footerline1">
+
+            <label for="email"></label>
+            <input type="email" id="email"  placeholder="Email adress" required /><br>
+            <hr class="footerline2">
+
+            <textarea name="text"  maxlength="500" placeholder="Enter text here" required /></textarea>
```
Comment (frank1738) [inline index.html:261]: - [ ] Kindly style your `get in touch` button to match the original design. Also, adjust the padding around the texts in the `<textarea/>` 

image

### Checkout the original design

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Form submission is working properly👍 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (hydaline-code) [reply index.html]: hi, thanks it's done!

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @hydaline-code    👋 
 
 # STATUS: APPROVED ✔️  🍾  🥳

 Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉

 
 
 ## Highlights
✔️  Proper use of Github features 💯 
✔️  well-crafted pull request 👍 
✔️ Linters are passing 👏 
 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._
 

 
 Cheers and Happy coding!👏👏👏
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**